### ISSUE TITLE
Word wrap width does not behave correctly

### ISSUE THREAD

**Issue by jrabek** [PhaserEditor2D/PhaserEditor2D-v3#135]
* Phaser Editor 2D Version: 3.15
* Operating System: MAc Big Sur

## Description
There are two issues with word wrap width.

1) When setting wordWrapWidth to 0, the generated typescript code is not correct:
```
textObject.setWordWrapWidth(textObject.style.wordWrapWidth, true);
```
`wordWrapWidth` can be a number or null which can not be passed to `setWordWrapWidth`.  Also it shouldn't be passed at all since it hasn't been set yet.

The value of `0` should just be used.

2) When setting wordWrapWidth to 1 to 24 the value in code isn't updated.  A value of 25 or higher must be used.

**Comment by PhaserEditor2D** [comment]
>     1When setting wordWrapWidth to 1 to 24 the value in code isn't updated.  A value of 25 or higher must be used.

Yes, I can see the error in the dev console: 
![image](https://user-images.githubusercontent.com/12187299/125314266-0a78df00-e304-11eb-9f99-19e0b682e2c5.png)

**Comment by PhaserEditor2D** [comment]
Phaser is checking if the width is enough for displaying one character. If it is not enough, it sends an error. I think it should just use the min-width needed, and never send that error.

But well.. I will handle it in the editor and later create an issue in the Phaser repo.

**Comment by jrabek** [comment]
Ah gotcha.  Yeah, the other issue is that `textObject.style.wordWrapWidth` is used when the width is 0 which typescript complains about since it can be undefined.  Seems like the codegen should allow for 0? Or something else?

**Comment by PhaserEditor2D** [comment]
> Ah gotcha. Yeah, the other issue is that `textObject.style.wordWrapWidth` is used when the width is 0 which typescript complains about since it can be undefined. Seems like the codegen should allow for 0? Or something else?

I will take a look. Thanks!

**Comment by PhaserEditor2D** [comment]
> Ah gotcha. Yeah, the other issue is that `textObject.style.wordWrapWidth` is used when the width is 0 which typescript complains about since it can be undefined. Seems like the codegen should allow for 0? Or something else?

This was fixed in v3.20.0: https://github.com/PhaserEditor2D/PhaserEditor2D-v3/releases/tag/v3.20.0
